src/libs/renderer/src/s_device.cpp

```diff
@@ -1435,13 +1435,9 @@ bool DX9RENDER::TextureLoad(int32_t t)
         return false;
     }
 
-    std::string_view name_sv(Textures[t].name, strlen(Textures[t].name));
-    auto has_resource_prefix =
-        std::distance(std::ranges::search(name_sv, "resource\\textures\\"sv, storm::detail::is_iequal{}).begin(),
-                      name_sv.begin()) == 0;
-    auto has_tx_postfix =
-        std::distance(std::ranges::search(name_sv, ".tx"sv, storm::detail::is_iequal{}).begin(),
-                      name_sv.end()) == 3;
+    tolwr(Textures[t].name);
```

Comment:
if we do this how would we use upper register texture name on case-sensetive filesystem?

Reply:
My `starts_with`/`ends_with` is case sensitive. So I just make `Textures[t].name` in low case to make sure that it will find "resource\\textures\\" or ".tx".
Reading in case sensetive filesystem will be in next PRs.

Reply:
the question is can it affect the system? originally, `Textures[t].name` contained a case-sensitive path
e.g. if the texture is being repeatedly requested but the matching search is case-sensitive, this would introduce superfluous resource allocation

Reply:
Changed. I hope this fix is good.